js/config.js

```diff
@@ -1,197 +1,128 @@
-        },
-    ],
-}
+            settings_modal: {
+                title: logo,
+                save_settings_btn: 'Save my choices',
+                accept_all_btn: 'Accept all',
+                reject_all_btn: 'Reject all',
+                close_btn_label: 'Close',
+                cookie_table_headers: [
+                    {col1: 'Name'},
+                    {col2: 'Domain'},
+                    {col3: 'Expiration'},
+                    {col4: 'Description'}
+                ],
+                blocks: [
+                    {
+                        title: 'Cookie usage and the Fitzwilliam website',
+                        description: 'The Fitzwilliam Museum uses cookies to ensure the basic functionalities of the website and to enhance your online experience. You can choose for each category to opt-in/out whenever you want. For more details relative to cookies and other sensitive data, please read the full <a href="#" class="cc-link">privacy policy</a>.'
+                    }, {
+                        title: 'Strictly necessary cookies',
+                        description: 'These cookies are essential for the proper functioning of my website. Without these cookies, the website would not work properly',
+                        toggle: {
+                            value: 'necessary',
+                            enabled: true,
+                            readonly: true          // cookie categories with readonly=true are all treated as "necessary cookies"
+                        }
+                    }, {
+                        title: 'Performance and Analytics cookies',
+                        description: 'These cookies allow the website to remember the choices you have made in the past',
+                        toggle: {
+                            value: 'analytics',     // there are no default categories => you specify them
+                            enabled: false,
+                            readonly: false
+                        },
+                        cookie_table: [
+                            {
+                                col1: '^_ga',
+                                col2: 'google.com',
+                                col3: '2 years',
+                                col4: 'description ...',
+                                is_regex: true
+                            },
+                            {
+                                col1: '_gid',
+                                col2: 'google.com',
+                                col3: '1 day',
+                                col4: 'description ...',
+                            }
+                        ]
+                    }, {
+                        title: 'Advertisement and Targeting cookies',
+                        description: 'These cookies collect information about how you use the website, which pages you visited and which links you clicked on. All of the data is anonymized and cannot be used to identify you',
```

Comment:
`anonymized` should be changed to British spelling `anonymised`

Reply:
See above https://github.com/FitzwilliamMuseum/fitz-main-website/blob/master/public/js/config.js#L114 - fitz content team to decide whether to just change here or all places?